SimPEG/EM/Static/DC/FieldsDC.py

```diff
@@ -116,9 +116,9 @@ def _e(self, phiSolution, srcList):
         """
             In HJ formulation e is not well-defined!!
             .. math::
-                \vec{e} = -\nabla \phi
+                \vec{e} = \rho \vec{j}
         """
-        return self.prob.MfI*self.prob.MfRhoI * self._j(phiSolution, srcList)
+        return self.prob.MfI*self.prob.MfRho * self._j(phiSolution, srcList)
```

Comment:
Thanks @micmitch! @sgkang: I am not sure if you updated this already on #590, but we can bring this in, along with the additional test and handle the merge after